src/WebJobs.Script.Host/Program.cs

```diff
@@ -21,14 +25,27 @@ public static void Main(string[] args)
                 rootPath = (string)args[0];
             }
 
-            var config = new ScriptHostConfiguration()
-            {
-                RootScriptPath = rootPath,
-                IsSelfHost = true
-            };
+            var host = new HostBuilder()
+                .ConfigureLogging(b =>
+                {
+                    b.SetMinimumLevel(LogLevel.Information);
+                    b.AddConsole();
+                })
+                .AddScriptHost(o =>
+                {
+                    o.ScriptPath = rootPath;
+                    o.LogPath = Path.Combine(Path.GetTempPath(), "functionshost");
+                    o.IsSelfHost = true;
+                })
+                .UseConsoleLifetime()
+                .Build();
+
+            Console.WriteLine("Starting host");
```

Comment:
Where will this be logged?

Reply:
This is in the console sample app. This won't be logged, just written to console.